roles/ceph-rgw-loadbalancer/templates/haproxy.cfg.j2

```diff
@@ -43,9 +43,8 @@ frontend rgw-frontend
 backend rgw-backend
     option forwardfor
     balance static-rr
-    option httpchk Get /
```

Comment:
Why remove the `httpchk` option here? It seems better to do a full HTTP check rather than just a TCP check, doesn't it?

Reply:
To (maybe) answer my own question, the `httpchk` above fails when checks are enabled, because `Get` isn't a valid HTTP verb. It works with this though:

```
    option httpchk GET /
```

Reply:
Ah, that makes more sense - thanks @BenoitKnecht. That would be a more appropriate fix. Will give it a shot and update this PR afterwards.